docs/workflows/sign-images.rst

```diff
@@ -298,3 +298,27 @@ It is possible to specify a single manifest identified by tag or a list of manif
 by proviging ``tags_list`` option to the call.
 Note that ``manifest lists`` are not signed, instead all the image manifests that manifest lists
 contain, are signed.
+
+Push signed images to the Pulp Registry
```

Comment:
push image signature to the pulp registry

Reply:
let me know what you think -not exactly sure this is the best place but it's convenient to add another section right below since it's about extensions api - 'serve signed images/image signature from the pulp registry.'
i'd cover in this section that  synced/pushed/or signed from within pulp signatures can be served by these extensions api and show an example of a GET call

Reply:
and also a note about containers registries.d file sigstore configuration on the client side